docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18097,6 +18097,284 @@
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.HDInsight/clusters"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.resourceProviderConnection",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.resourceProviderConnection",
+                        "notEquals": "Outbound"
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.privateLink",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.privateLink",
+                        "notEquals": "Enabled"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-HdInsight-NetworkProperties"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-HdInsight-Sku",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforces certain sku's for hd insight clusters.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Budget"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              },
+              "allowedVmSizes": {
+                "type": "Array",
+                "metadata": {
+                  "displayName": "Allowed VM Sizes",
+                  "description": "Specifies the allowed vm sizes for HDInsight"
+                },
+                "defaultValue": [
+                  "Standard_A1_v2",
```

Comment:
Should 'A' series VMs be suggested as these dont allow reservations etc?

Reply:
By default, it may makes sense to allow all SKUs and then the customer can decide which ones they want to deny?